samples/client/echo_api/java/apache-httpclient/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-6.4.0-SNAPSHOT
\ No newline at end of file
+6.3.0-SNAPSHOT
```

Comment:
looks like your change is not based on the latest master

Reply:
Huh, weird. I synced my fork with the OpenAPITools main branch, rebased my feature branch, and then re-ran `./bin/generate-samples.sh ` with the same result. I'll try to resolve this.